NEWS.rst

```diff
@@ -21,6 +21,9 @@ where appropriate by ``collections.defaultsdict``.
 The ``Bio.motifs.jaspar.db`` now returns ``tf_family`` and ``tf_class`` as a
 string array since the JASPAR 2018 release.
 
+The Local Composition Complexity functions from ``Bio.SeqUtils`` now uses
+base 4 log instead of 2.
+
```

Comment:
Maybe add something like: As per the original reference XXX.

Reply:
adding this info now

Reply:
How about this:

```
The Local Composition Complexity functions from ``Bio.SeqUtils`` now uses
base 4 log instead of 2 as stated in the original reference Konopka (2005),
Sequence Complexity and Composition. https://doi.org/10.1038/npg.els.0005260
```

See https://www.crossref.org/display-guidelines/ and our pre-commit check.

Reply:
I can change to this DOI URL type. And I am using the pre-commit check with the hook.